CHARTER.md

```diff
@@ -0,0 +1,238 @@
+
+Secondary Concerns
+- Hypermedia
+- Generating JSON Schema
+- Using JSON Schema to generate
+  - Code (including types or classes)
+  - UI (including forms)
+  - Databases
+- Relational validation
+- Vocabularies registry
+
+### 1.2: Out-of-Scope
+While the JSON Schema project has no control over the specifications it uses (such as JSON and IRI), nor specifications that use JSON Schema (such as OpenAPI and AsyncAPI), the JSON Schema project has, and continues to, provide expert opinion, advice, and support, as time allows. We recognize the importance and value of the ecosystem surrounding JSON Schema, and we encourage those developing standards and tooling to reach out at any time.
+
+## 2: Relationship with OpenJS Foundation CPC.
+Most large, complex open source communities have both a business and a technical governance model. Technical leadership for the projects within the OpenJS Foundation is delegated to the projects through their project charters by the OpenJS Cross Project Council (CPC). In the case of the JSON Schema project, it is delegated to the JSON Schema Technical Steering Committee ("TSC").
+
+### 2.1 Other Formal Project Relationships
+Section Intentionally Left Blank
+
+## 3: JSON Schema Org Governing Body (TSC)
+The TSC is initially established from the observed major contributors who are currently active and in good standing.
+
+There is no maximum TSC membership size. The TSC must have a minimum of four members.
+
+Changes to TSC membership should be posted in the agenda, and may be suggested as any other agenda item.
+
+TSC memberships are not time-limited.
+
+While the project is not looking to obtain "Impact" project status within the OpenJS Foundation, there is no requirement set out to limit the number of TSC members by employer. It is in some cases considered difficult or even unhelpful for the project to limit the number or percentage of TSC members by employer (especially when an employer has employed individuals already active in the community to work exclusively on the open source project). While at this time there are no limits on TSC membership by employer, the TSC will strive to keep to at least less than 50%, ideally 33% (One third, one in three). The TSC will re-evaluate this specific clause at least every six months, and aim to revise it within one year to meet the "no more than 1/3 employer member affiliation" mandate.
+
+TSC members are expected to regularly participate in TSC activities.
+
+The TSC will meet regularly using virtual conferencing tools. The meeting will be directed by the TSC Chairperson(s). Responsibility for directing individual meetings may be delegated by a TSC Chairperson to any other TSC member. Minutes or an appropriate recording will be taken and made available to the community through accessible public postings.
+
+The TSC may, at its discretion, invite any number of non-voting observers to participate in the public portion of TSC discussions and meetings.
```

Comment:
That's the intent! Public as much as possible!
I note that the CPC has a private portion of the call on occasion, for whatever reason.

Reply:
Whereby I meant that the session should be public by default (with an agenda and videoconf link made available upfront), not that it is private by default and that the TSC may decide to open it.

Reply:
I think this sentence is correct. It should be specified somewhere else that there might a private portion of the meeting.

Reply:
> I think this sentence is correct. It should be specified somewhere else that there might a private portion of the meeting.

Well, but the problem of making allowing access to the TSC meeting at the discretion of the TSC, is that this makes the meeting private by default, and requires action by the TSC to make it public.

Reply:
It doesn't say that. It says the TSC may invite people to the public portion of the meeting. You're reading into it something which isn't there.

Technically, as it is public, and the details of how to access it will be public, anyone could invite anyone. But I wanted to make it explicit that TSC may invite guests.

I don't recall specifically where I borrowed the phrasing from, but I agree it is not clear and can be improved.

Reply:
> You're reading into it something which isn't there.

I dont doubt your intentions. I’m stating that the text doesn’t match them. ;)